plutus-core/plutus-core/src/PlutusCore/Core/Type.hs

```diff
@@ -118,8 +118,13 @@ data Term tyname name uni fun ann
     | Constant ann (Some (ValueOf uni)) -- ^ constants
     | Builtin ann fun -- ^ builtin functions
     | Error ann (Type tyname uni ann) -- ^ fail with error
-    deriving stock (Show, Functor, Generic)
-    deriving anyclass (NFData)
+    deriving stock (Functor, Generic)
+
+deriving stock instance (Show tyname, Show name, GShow uni, Everywhere uni Show, Show fun, Show ann, Closed uni)
```

Comment:
:cry:

Reply:
Yeah, it really sucks. But perhaps we could introduce some aliases in order not to write this whole thing our every time. Maybe some generic one that works for every constraint (we'll only need to special-case things like `GShow`). Not sure if it's worth it, though.